arrows/vtk/applets/color_mesh.cxx

```diff
@@ -64,6 +65,31 @@ bool check_config(kv::config_block_sptr config)
     validate_required_input_file("video_source", *config, main_logger)
     && config_valid;
 
+  if (config->has_value("output_mesh"))
+  {
+    std::string output_mesh_ =
```

Comment:
style hint:  The `_` suffix is often used to distinguish class member variables from other local variables, so it probably shouldn't be used here.  We are not super consistent about this.  Some code in KWIVER uses a `m_` prefix instead of a `_` suffix for this purpose.